docs/source/api.rst

```diff
@@ -159,6 +159,18 @@ Chalice
         entire lambda function name.  This parameter is optional.  If it is
         not provided, the name of the python function will be used.
 
+   .. method:: lambda_function(name=None)
+
+      Create a pure lambda function that's not connected to anything.
+
+      See :doc:`purelambda` for more information.
+
+      :param name: The name of the function to use.  This name is combined
```

Comment:
Can you say how it is combined. Like: {app}-{stage}-{name} notation or give an example. Or do that in the extended `purelambda` docs.